### ISSUE TITLE
It seems that tabula.read_pdf_with_template always guesses the area

### ISSUE THREAD

**Issue by papagala** [chezou/tabula-py#124]
I am currently specifying a template, but it seems to be ignored. My template is just the entire page of the pdf and I cannot seem to extract all the information from Python. When I use the same template from Tabula app, I am able to extract all the information from the PDF. When I try the same from tabula-py, I only get the top infromation. 

# Environment

Write and check your environment. Please paste outputs of specific commands if required.

-Python version:
    3.6.4 |Anaconda, Inc.| (default, Jan 16 2018, 10:22:32) [MSC v.1900 64 bit (AMD64)]
Java version:
    java version "1.8.0_171"
Java(TM) SE Runtime Environment (build 1.8.0_171-b11)
Java HotSpot(TM) 64-Bit Server VM (build 25.171-b11, mixed mode)
tabula-py version: 1.3.1
platform: Windows-7-6.1.7601-SP1
machine='AMD64', processor='Intel64 Family 6 Model 78 Stepping 3, GenuineIntel')
linux_distribution: ('MSYS_NT-6.1', '2.10.0', '')
mac_ver: ('', ('', '', ''), '')

# What did you do when you faced the problem?

//write here

## Example code:

Here is the code I'm using.

import tabula

df=tabula.read_pdf_with_template('distro-pdfs/small/document-page1.pdf','templates/full.tabula-template.json',pandas_options={'header': None},guess=False)

## Output:

0	1	2	3	4	5	6	7	8
CUPON	PORCENTAJE MAXIMO	VENCIDOS F. PAGO ANTERIOR	VIGENTES FECHA DE CORTE	VENCIDA	%		CARTERA VENCIDA	
1	4%	0.00	3,407,703,808.60	136,308,152.34		0.00%		-
2	8%	0.00	3,407,703,808.60	272,616,304.69		0.00%		-
3	12%	349,015.34	3,407,703,808.60	408,924,457.03		0.01%		-
4	16%	2,518,825.34	3,407,703,808.60	545,232,609.38		0.07%		-
7	16%	24,435,356.14	3,407,703,808.60	545,232,609.38		0.72%		-

## What did you intend it to be?

	0	1	2	3	4	5	6	7
0		Evento de Cartera Vencida Acumulada						
1				IMPORTE MAXIMO				
2		CREDITOS HIPOTECARIOS		CARTERA			IMPORTE POR EVENTO	
3	CUPON	PORCENTAJE MAXIMO VENCIDOS F. PAGO ANTERIOR VIGENTES FECHA DE CORTE		VENCIDA	%		CARTERA VENCIDA	
4	1	4% 0.00 3,407,703,808.60		136,308,152.34		0.00%		-
5	2	8% 0.00 3,407,703,808.60		272,616,304.69		0.00%		-
6	3	12% 349,015.34 3,407,703,808.60		408,924,457.03		0.01%		-
7	4	16% 2,518,825.34 3,407,703,808.60		545,232,609.38		0.07%		-
8	7	16% 24,435,356.14 3,407,703,808.60		545,232,609.38		0.72%		-
9		Evento de Cartera Vencida Acumulada						
10		$ Evento Cartera Vencida 0.00						
11		Saldo Disponible 268,260,832.96						
12		$ Evento Cartera Vencida Fecha de Pago 0.00						
13		SIP CB's Serie A > SIP Creditos Hipotecarios en la Fecha de Pago						
14		SIP CB's Serie A1 1,750,142,969.48						
15		SIP Creditos Hipotecarios Vigentes en la Fecha de Pago 2,429,205,681.71						
16		SIP CB's Serie A > SIP Creditos Hipotecarios en la Fecha de Pago NO						
17		Porcentaje Serie B y Constancia						
18				Constancia subordinada				
19		(Saldo de la Serie / Cartera Vigente) Serie A preferente						
20		Porcentaje  inicial 78.86% 7.50%		13.64%				
21		Porcentaje después amortización 64.50% 7.50%						
22		Porcentaje objetivo 64.50% 7.50%		182,190,426.13				
23		30,549,416.38 UDISSaldo objetivo para aforo objetivo UDIS262,724,980.89 UDIS						
24		Amortización necesaria para alcanzar el						
25		30,736,357.56 UDIS 3,573,995.06 UDIS		10,297,068.90				
26		porcentaje objetivo en UDIS						
27		1,566,837,664.70 182,190,426.13		-				
28		Saldo objetivo para porcentaje objetivo Pesos						
29		Amortización máxima para alcanzar el						
30		183,305,304.78 21,314,570.28						
31		porcentaje objetivo Pesos						
32		2,429,205,681.71						
33		Saldo  cartera hipotecaria vigente anterior						
34		Sobrecolaterización final						
35		Sobrecolateral Serie A 35.50%						

-link pdf

https://s3.amazonaws.com/samplesinfonavit/document-page1.pdf

-link template

https://s3.amazonaws.com/samplesinfonavit/full.tabula-template.json

**Comment by chezou** [comment]
Thanks for reporting. As far as I read the template, it determines `extraction_method` as `guess` i.e. guess option will be used.

```json
[{"page":1,"extraction_method":"guess","x1":1.1475,"x2":612.3825,"y1":1.1475,"y2":789.8625000000001,"width":611.235,"height":788.715}]
```

You have to change `extraction_method` on tabula, but I don't know how.

btw, pandas is hard to handle with different column number tables at once. It's natural to have 4 entities in the array of JSON.

**Comment by papagala** [comment]
Hey @chezou, thanks for solving this so quickly. I found something which fixed my problem ("extraction_method":"stream"):

[{"page":1,**"extraction_method":"stream"**,"x1":1.1475,"x2":612.3825,"y1":1.1475,"y2":789.8625000000001,"width":611.235,"height":788.715}]

If I understood correctly " is hard to handle with different column number tables at once. It's natural to have 4 entities in the array of JSON", you mean I should not use my approach of using the entire page as template?

The reason I did this, is that this made my extraction very robust to small changes of coordinates in the tables. It has been very challenging indeed, but I have gained a new level of extraction which is not susceptible to small changes.  Thanks again!
